packages/liferay-npm-scripts/src/presets/standard/index.js

```diff
@@ -16,7 +16,8 @@ module.exports = {
 	build: {
 		dependencies: [...clay, ...liferay, ...metal],
 		input: 'src/main/resources/META-INF/resources',
-		output: 'build/node/packageRunBuild/resources'
+		output: 'build/node/packageRunBuild/resources',
+		temp: 'build/npmscripts'
```

Comment:
Are there any special considerations with adding a new subfolder under `build/`? ie. is there any tooling anywhere else that might be affected? Also, wondering why this isn't under `build/node`, seeing as it is part of the "node build" (although I don't think we can push it further down under `build/node/packageRunBuild` because I suspect our Gradle SDK ends up just copying everything out of there).

Reply:
I didn't want to put it under `node` because that is controlled by our gradle plugin and it used to put more things (in addition to `packageRunBuild`) so I wanted to minimize the possibility of a conflict.

There's no standard about what may be put in `build`, but it looks reasonable to have one folder per _logic unit_ so that they don't collide.